pkg/master/master.go

```diff
@@ -87,22 +88,25 @@ const (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	DatabaseStorage tools.StorageInterface
-	EventTTL        time.Duration
-	MinionRegexp    string
-	KubeletClient   client.KubeletClient
+	DatabaseStorage    tools.StorageInterface
+	ExpDatabaseStorage tools.StorageInterface
```

Comment:
No, it needs to be different so that the right Codec is used. Right now these are the same but when they become separate groups it will matter.